src/reducers/classifierReducer.js

```diff
@@ -96,7 +95,8 @@ export default function classifier(state=InitialClassifier, action) {
             if (subject) {
                 return { ...state, subject }
             } else {
-                return { ...state, workflowOutOfSubjects: true }
+                // If the user has seen every subject, display a random one they have seen
+                return { ...state, subject: subjectList[Math.floor(Math.random() * subjectList.length)] }
```

Comment:
Doesn't look like these subjects possess the "already seen" banner the second time around.